### DISCUSSION TITLE
Base Directory

### DISCUSSION THREAD

**Discussion by adhiduzz** [spencerwooo/onedrive-vercel-index#664] [General]
Hi
can i produce multiple 'base directory'?

tried to write like this, no luck

// The folder that you are to share publicly with onedrive-vercel-index. Use '/' if you want to share your root folder.
  baseDirectory: '/testing1','/testing2'

**Comment by myl7** [comment]
Currently no, and to support it a large code refactoring should be required, which does not seem to be an easy thing.

**Reply by adhiduzz** [reply]
ooh ok
thanks for response
